dbt_semantic_interfaces/objects/semantic_manifest.py

```diff
@@ -10,3 +12,11 @@ class SemanticManifest(HashableBaseModel):
 
     semantic_models: List[SemanticModel]
     metrics: List[Metric]
+
+    @property
+    def interfaces_version(self) -> str:
+        """Returns the version of the dbt_semantic_interfaces package that generated this manifest."""
+        try:
+            return version("dbt_semantic_interfaces")
+        except PackageNotFoundError:
+            return "dbt_semantic_interfaces is not installed"
```

Comment:
We should instead do a pydantic validator like the following
```python
    interfaces_version: str = ""    

    @validator('interfaces_version', always=True)
    @classmethod
    def __create_default_interfaces_version(cls, value: Any, values: Any) -> str:  # type: ignore[misc]
        if value:
            return value
        return version("dbt_semantic_interfaces")
```

Reply:
Note I dropped the `return "dbt_semantic_interfaces is not installed"`. If we do that return then the version would become `"dbt_semantic_interfaces is not installed"` and honestly I think we'd prefer it just break if the package isn't installed.